src/config/mod.rs

```diff
@@ -166,6 +166,7 @@ impl Config {
 pub enum ProtocolVersion {
     Reserved = 0,
     Mls10 = 1,
+    Mls10Draft12 = 255, // pre RFC version
```

Comment:
We are on draft 11 right now, just in case. I think we need room for ~10 pre-RFC draft versions, so maybe start at 200?

Reply:
Yeah I figured by the time we need this we are on 12 ;)
Using an enum for versions is just not a great idea... But let's start at 200.